Firmware/RTK_Surveyor/WiFi.ino

```diff
@@ -50,6 +50,8 @@ static const int WIFI_CONNECTION_TIMEOUT = 8 * 1000;  //Milliseconds
 //Interval to use when displaying the IP address
 static const int WIFI_IP_ADDRESS_DISPLAY_INTERVAL = 12 * 1000;  //Milliseconds
 
+#define WIFI_MAX_NMEA_CLIENTS         4
+
```

Comment:
First, I'm really glad to see NMEA over TCP as this is the key feature to make this work with Vespucci, an OSM editor for Android.

What would be great is some magic token that can be put in the client list that is "connect to the router".  The point is that with the phone in hotspot mode, the address varies every reboot (that's how android is these days) and thus configuring a fixed address isn't going to work.  With vespucci set up to listen on a port,and the RTK Express set to connect to the router (which is the phone) on that port, it should just work.

FWIW, Vespucci has the ability to connect to an address/port and listen for NMEA, or to listen on a port, accept connections, and then receive NMEA.

Reply:
What you are looking for is in PR [#309](https://github.com/sparkfun/SparkFun_RTK_Firmware/pull/309).